### PR TITLE
Filter saving and deleting

### PR DESCRIPTION
**Pull request by sabineabele**

### CODE REVIEW

File: resources/assets/stylesheets/layout/filter.scss [hunk 1]
```diff
@@ -143,5 +143,35 @@
...
+  }
+}
+
+.saved-filters{
+  flex: 2 0 0;
+}
+.saved-filter {
+  position: relative;
+  float: right;
+  margin-right: steps(0.75);
```
Comment (exabyssus) [inline resources/assets/stylesheets/layout/filter.scss]: Minimal step division should be 0.5. Please use 'px' units if.

File: src/Admin/Traits/Crudify.php [hunk 2]
```diff
@@ -106,6 +107,8 @@ protected function buildGrid(Model $model)
         $grid = new Grid($model);
         $grid->setModule($this->module());
         $grid->setRenderer(new Grid\Builder($grid));
+        $grid->setFilterManager(app(FilterManager::class)); // TODO: Use proper DI
```
Comment (exabyssus) [inline src/Admin/Traits/Crudify.php]: If DI is impossible, remove the TODO comment

Comment (exabyssus) [reply src/Admin/Traits/Crudify.php]: Otherwise it should be implemented

File: resources/assets/js/include/filter.js [hunk 3]
```diff
@@ -21,6 +18,13 @@ jQuery(document).ready(function() {
         openCloseFilter();
     });
 
+    function addSerchToFilter() {
```
Comment (cliche23) [inline resources/assets/js/include/filter.js]: Typo in function name

**Pull request merged**